models/product.model.js

```diff
@@ -0,0 +1,63 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const ProductSchema = new Schema({
+    name : {
+        type: String,
+        required: 'Product name is required'
+    },
+    
+    image : {
+        type: String
+    },
+    
+    price : {
+        type: Number,
+        required: 'Product price is required'
+    },
+
+    category : {
+        type: String,
+        required: 'Product category is required'
+    },
+    
+    brand : {
+        type: String
+    },
+    
+    inStock : {
+        type: Boolean,
+        required: 'Product is instock information is required'
+    },
+    
+    fastDelivery : {
+        type: Boolean
```

Comment:
default False kardo.. if it is not required, and true false value hai, toh I think it's a good habit to add a default value